src/plugins/gs-plugin-external-apps.c

```diff
@@ -393,7 +416,7 @@ build_runtime (GsPlugin *plugin,
 	const char *runtime_name = gs_app_get_id_no_prefix (runtime);
 	const char *runtime_url = gs_app_get_metadata_item (runtime,
 							    METADATA_URL);
-	const char *runtime_type = gs_app_get_metadata_item(runtime,
+	const char *runtime_type = gs_app_get_metadata_item (runtime,
 							    METADATA_TYPE);
```

Comment:
This fix doesn't belong in this commit but rather in the one where Cosimo commented on.
Also, please verify that METADATA_TYPE is well-aligned after the new space introduction (I am not sure if it's right or wrong here due to the mix of tabs and spaces that g-s uses..).